doc/pilight-flash.1

```diff
@@ -0,0 +1,66 @@
+.TH pilight-flash 1 "3 December 2016" "7.0" "pilight 7.0"
+.SH NAME
+pilight-flash \- Flash hardware with pilight firmware
+.SH SYNOPSIS
+.B pilight-flash \fI--file FIRMWARE_FILE\fR
+.br
+.B pilight-control [\fIOPTION\fR]... \fI--file FIRMWARE_FILE\fR
+.SH DESCRIPTION
+.B pilight-control
+is a tool to flash pilight firmware to hardware such as ATTiny or Arduino 
+chips. 
+.PP
+In case of ATTiny firmware, it is responsible for pre-filtering streams and 
+removing garbage data from the receiver to reach the pilight-daemon. This 
+lowers CPU usage by the pilight daemon greatly, since it will not attempt 
+to continuously interpret received garbage singals as valid protocols. 
+Currently supported are the ATTiny25, ATTiny45, and ATTiny85.
+.PP
+The Arduino firmware is necessary for using pilight with senders and 
+receivers on non-GPIO capable hardware (most desktops, etc.) The flashed 
```

Comment:
Also not entirely true because you can also use the `Atmel` chips on GPIO capable hardware as long as it has USB support. Again Arduino should say Atmel.